docs/architecture/adr-007-specialization-groups.md

```diff
@@ -0,0 +1,161 @@
+# ADR 007: Specialization Groups
+
+## Changelog
+
+- 2019 Jul 31 : Initial Conception
+
+## Context
+
+This idea was first conceived of in order to fulfill the use case of the
+creation of a decentralized Computer Emergency Response Team (CERT), Whose
+members would be elected by the Cosmos Community and would fulfill the role of
+coordinating the Cosmos community under an emergency situations. This thinking
+can be further abstracted into the conception of "blockchain specialization
+groups". 
+
+The creation of these groups are the beginning of specialization capabilities
+within a wider blockchain community which could be used to enable a certain
+level of delegated responsibilities. Examples of specialization which could be
+beneficial to a blockchain community include: code auditing, emergency response,
```

Comment:
Do you have an example of emergency response? Also an example to show where a CERT team could handle an emergency more effectively than direct governance vote would be useful in providing context.

Reply:
I've added more discussion (but not here) to the Context section of ADR 008 (in this PR) which should fully describe what you're looking for